sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsClientJavaDocCodeSnippets.java

```diff
@@ -123,50 +145,52 @@ public void detectLanguageStringListWithOptions() {
             "This is written in English",
             "Este es un document escrito en Español."
```

Comment:
looks like this string is copied in multiple places. Make sure to replace `document` to `documento`